storybook/mocks/wallet.ts

```diff
@@ -36,7 +37,7 @@ const walletMeta: WalletMeta = {
   walletImplementationId: 'haskell-shelley-24',
 }
 
-const wallet: YoroiWallet = {
+const wallet: YoroiWallet & Record<string, unknown> = {
```

Comment:
Why is the `& Record<string, unknown>` needed here?

Reply:
leftovers from previous test strategy